app/code/core/Mage/Payment/Model/Method/Cc.php

```diff
@@ -283,117 +278,4 @@ public function isAvailable($quote = null)
         return $this->getConfigData('cctypes', ($quote ? $quote->getStoreId() : null))
             && parent::isAvailable($quote);
     }
-
-    /**
-     * Whether centinel service is enabled
-     *
-     * @return bool
-     */
-    public function getIsCentinelValidationEnabled()
-    {
-        return Mage::getConfig()->getNode('modules/Mage_Centinel') !== false && $this->getConfigData('centinel') == 1;
-    }
-
-    /**
-     * Instantiate centinel validator model
-     *
-     * @return Mage_Centinel_Model_Service
-     */
-    public function getCentinelValidator()
-    {
-        $validator = Mage::getSingleton('centinel/service');
-        $validator
-            ->setIsModeStrict($this->getConfigData('centinel_is_mode_strict'))
-            ->setCustomApiEndpointUrl($this->getConfigData('centinel_api_url'))
-            ->setStore($this->getStore())
-            ->setIsPlaceOrder($this->_isPlaceOrder());
-        return $validator;
-    }
-
-    /**
-     * Return data for Centinel validation
-     *
-     * @return Varien_Object
-     */
-    public function getCentinelValidationData()
-    {
-        $info = $this->getInfoInstance();
-        $params = new Varien_Object();
-        $params
-            ->setPaymentMethodCode($this->getCode())
-            ->setCardType($info->getCcType())
-            ->setCardNumber($info->getCcNumber())
-            ->setCardExpMonth($info->getCcExpMonth())
-            ->setCardExpYear($info->getCcExpYear())
-            ->setAmount($this->_getAmount())
-            ->setCurrencyCode($this->_getCurrencyCode())
-            ->setOrderNumber($this->_getOrderId());
-        return $params;
-    }
-
-    /**
-     * Order increment ID getter (either real from order or a reserved from quote)
-     *
-     * @return string
-     */
-    private function _getOrderId()
```

Comment:
This and following functions don't seem to have anything to do with Centinel. Is it ok to delete?

Reply:
true, I didn't remove them at first but phpstan started complaining, you can see it here:
https://github.com/OpenMage/magento-lts/actions/runs/8931953986/job/24534948827

because those methods are private and not used so they should be un-usable by anybody

I think